### PR TITLE
[Debug] Trigger deprecation notices when using PHP7 reserved class names

### PR DESCRIPTION
**Pull request by nicolas-grekas**
| Q | A |
| --- | --- |
| Bug fix? | yes |
| New feature? | no |
| BC breaks? | no |
| Deprecations? | no |
| Tests pass? | yes |
| Fixed tickets | - |
| License | MIT |
| Doc PR | - |

### CODE REVIEW

File: src/Symfony/Component/Debug/Tests/DebugClassLoaderTest.php [hunk 1]
```diff
@@ -249,6 +271,8 @@ public function findFile($class)
             eval('namespace Test\\'.__NAMESPACE__.'; class DeprecatedParentClass extends \\'.__NAMESPACE__.'\Fixtures\DeprecatedClass {}');
         } elseif ('Test\\'.__NAMESPACE__.'\DeprecatedInterfaceClass' === $class) {
             eval('namespace Test\\'.__NAMESPACE__.'; class DeprecatedInterfaceClass implements \\'.__NAMESPACE__.'\Fixtures\DeprecatedInterface {}');
+        } elseif ('Test\\'.__NAMESPACE__.'\Float' === $class) {
+            eval('namespace Test\\'.__NAMESPACE__.'; class Float {}');
```
Comment (xabbuh) [inline src/Symfony/Component/Debug/Tests/DebugClassLoaderTest.php]: It seems that we [cannot execute this on PHP 7](https://travis-ci.org/symfony/symfony/jobs/60078716#L491).

Comment (nicolas-grekas) [inline src/Symfony/Component/Debug/Tests/DebugClassLoaderTest.php]: Fixed by skipping the test on PHP 7.0

**Comment by xabbuh** [comment]
:+1:

**Comment by fabpot** [comment]
Thank you @nicolas-grekas.

**Pull request merged**